Formula/imagemagick.rb

```diff
@@ -126,7 +122,8 @@ def install
     args << "--with-quantum-depth=#{quantum_depth}" if quantum_depth
 
     # versioned stuff in main tree is pointless for us
-    inreplace "configure", "${PACKAGE_NAME}-${PACKAGE_VERSION}", "${PACKAGE_NAME}"
+    inreplace "configure",
```

Comment:
Aligning under the ":" was definitely his idea https://github.com/Homebrew/homebrew-core/pull/3827#discussion_r74633980 :D